packages/react/src/primitives/Checkbox/useCheckbox.ts

```diff
@@ -4,8 +4,8 @@ import { isFunction } from '../shared/utils';
 import { UseCheckbox } from '../types/checkbox';
 
 export const useCheckbox = (
-  initialChecked: boolean,
-  onChangeProp: React.ChangeEventHandler
+  initialChecked: boolean | undefined,
+  onChangeProp: React.ChangeEventHandler | undefined
```

Comment:
What do you think of making this optional instead of allowing `undefined`?

Reply:
I'm good with that, will update